src/main/java/cubicchunks/server/PlayerCubeMap.java

```diff
@@ -523,9 +531,10 @@ public final void setPlayerViewRadius(int newViewDistance) {
 
 			long playerAddress = getAddress(playerCubeX, playerCubeY, playerCubeZ);
 
-			if (newViewDistance > oldViewDistance) {
+			// One of the view distances increased
+			if (newHorizontalViewDistance > oldHorizontalViewDistance || newVerticalViewDistance > oldVerticalViewDistance) {
```

Comment:
I don't think it's 100% correct. It will work in practice because only one of them can change at a time, but if both of them change at once in opposite directions, things will break. At least add comment explaining why it works and what are assumptions. Or throw exception if both of them change at once in opposite direction.